src/ec/packages/ec-component-expandable/expandable.story.js

```diff
@@ -0,0 +1,57 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, radios, text } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import demoData from './demo/data';
+
+import expandable from './expandable.html.twig';
+import notes from './docs/expandable.md';
+
+storiesOf('Components/Expandable', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'default',
+    () => {
+      demoData.button.icon.path = defaultSprite;
+      demoData.label_expanded = text('Label Expanded', demoData.labelExpanded);
+      demoData.label_collapsed = text(
+        'Label Collapsed',
+        demoData.labelCollapsed
+      );
+      demoData.content = text('Content', demoData.content);
+      demoData.expanded = radios(
```

Comment:
as the behavior is not fully functional (no label change, no changes of radio button if we click on the expandable component itself), I think that there is no need to display this knobs.
The boolean behind it is still useful in the twig template, but having it as a knob may raise questions.

Reply:
Agree, changed.